Makefile

```diff
@@ -13,6 +15,12 @@ generate:
 	go generate $(shell go list ./...)
 	$(MAKE) format
 
+generate-from-swagger: clean
+	cp $(shell go list -m -f={{.Dir}} github.com/filanov/bm-inventory)/swagger.yaml swagger.yaml
```

Comment:
Why do this magic?
Is this required for not keeping the generated code in github?

Reply:
https://golang.org/doc/articles/go_command.html
"check those generated source files into your repository. This is more work for you, the package author, but it is significantly less work for your users, who can use "go get" without needing to obtain and build any additional tools."

Reply:
@ronniel1 That's one way how we can ease our PRs and conflicts when forgetting to push swagger or matching generate files (already happened).
We are not an open source project with users who use us with "go get bm-inventory".

Reply:
I think we should strive to be an opensource project and allow users to go get bm-inventory.
Pretty easy to fail PR in case it's missing generated files.